iothub/service/src/Configurations/ConfigurationContent.cs

```diff
@@ -12,16 +12,28 @@ namespace Microsoft.Azure.Devices
     public class ConfigurationContent
     {
         /// <summary>
-        /// Gets or sets Module Configurations
+        /// Gets or sets the configurations to be applied to the Edge agent.
         /// </summary>
+        /// <remarks>
+        /// For more information on this field, see <see href="https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2020-11#create-a-deployment-manifest">this documentation</see>.
+        /// </remarks>
         [JsonProperty(PropertyName = "modulesContent")]
 #pragma warning disable CA2227 // Collection properties should be read only
         public IDictionary<string, IDictionary<string, object>> ModulesContent { get; set; }
 
 #pragma warning restore CA2227 // Collection properties should be read only
 
         /// <summary>
-        /// Gets or sets Module Configurations
+        /// Gets or sets the configurations to be applied on device modules
+        /// </summary>
+        [JsonProperty(PropertyName = "moduleContent")]
+#pragma warning disable CA2227 // Collection properties should be read only
```

Comment:
Do these have any benefits over the attribute version of warning suppression? As far as I know using the preprocessor directives and the using the suppression attribute are alternatives. Attributes read and look much cleaner specially for property warning suppressions.